src/vcpkg/commands.ci.cpp

```diff
@@ -438,26 +470,19 @@ namespace vcpkg::Commands::CI
 
         IBinaryProvider& binaryprovider = binaryproviderStorage ? *binaryproviderStorage : null_binary_provider();
 
-        const ParsedArguments options = args.parse_arguments(COMMAND_STRUCTURE);
-        auto& settings = options.settings;
-
-        std::set<std::string> exclusions_set;
-        auto it_exclusions = settings.find(OPTION_EXCLUDE);
-        if (it_exclusions != settings.end())
+        if (args.command_arguments.size() != 1)
         {
-            auto exclusions = Strings::split(it_exclusions->second, ',');
-            exclusions_set.insert(exclusions.begin(), exclusions.end());
+            Checks::unreachable(VCPKG_LINE_INFO); // this should not be possible at this place
```

Comment:
```suggestion
            Checks::unreachable(VCPKG_LINE_INFO);
```
The comment is repeating the definition of `unreachable`